relativedeltafield/__init__.py

```diff
@@ -134,18 +197,21 @@ def get_db_prep_value(self, value, connection, prepared=False):
 	# We can't simply replace or remove PsycoPg2's parser, because
 	# that would mess with any existing Django DurationFields, since
 	# Django assumes PsycoPg2 returns pre-parsed datetime.timedeltas.
+	# 	def select_format(self, compiler, sql, params):
+	# 		fmt = 'to_char(%s, \'PYYYY"Y"MM"M"DD"DT"HH24"H"MI"M"SS.US"S"\')' % sql
+	# 		return fmt, params
+
 	def select_format(self, compiler, sql, params):
-		fmt = 'to_char(%s, \'PYYYY"Y"MM"M"DD"DT"HH24"H"MI"M"SS.US"S"\')' % sql
+		if compiler.connection.vendor == 'postgresql3':
+			fmt = 'to_char(%s, \'PYYYY"Y"MM"M"DD"DT"HH24"H"MI"M"SS.US"S"\')' % sql
+		else:
+			fmt = sql
 		return fmt, params
 
-	if django.VERSION < (2,):
```

Comment:
I'm okay with dropping older Django support (older versions are untested anyway), but let's do it in a separate commit. This is completely unrelated to the django admin support.

Reply:
In reality I just simplified the if: there are no differences in the two if branches if we just leave one signature with an optional (and unused) context argument

Reply:
hm, I don't remember.. maybe this gave a deprecation warning before. But you're right.